app-former-showcase/app-former-webapp/pom.xml

```diff
@@ -50,6 +50,18 @@
     </dependency>
 
     <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-widgets-core-client</artifactId>
+      <scope>provided</scope>
```

Comment:
@jsoltes They are only needed at compilation time of the webapp; when GWT needs the "client-side" sources to generate the JavaScript. They are customarily set to ```provided``` scope so they are not bundled in the WAR.